doc/source/api-key-creation.rst

```diff
@@ -0,0 +1,37 @@
+.. _api-key-creation:
+
+====================
+API Key Creation
+====================
+
+API keys are used to authenticate requests made using the Conduce REST API.  API keys are created by the user and should be protected like passwords.  Anyone with an API key can make requests to Conduce and perform operations as the user who requested the key.
+
+Generally, there are three methods used to authenticate Conduce REST requests:
+
+**username/password**
+    used to establish a session
+**session cookie**
+    returned from a user name password login.  Used to continue a session started with a username/password login.
+**API key**
+    a unique string generated upon request that can be used to authenticate requests.
+
+In order to generate an API key, you must have a Conduce account.  Contact support@conduce.com if you do not already have an account.  Once your account is registered, you may create an API key.  It is easiest to do this using the Conduce command line utility.  Once you have :ref:`installed the Conduce Python API <https://github.com/ConduceInc/conduce-python-api>` execute the following command from your shell prompt::
+
+    conduce-api create-api-key --user=new_user@conduce.com --host=app.conduce.com
```

Comment:
You should explicitly tell the reader to replace this email address with the email address they used to create their account

Reply:
It would be nice if we could find a way to prevent the user from specifying the host.  A user will only interact with a single host, so this is just wasteful for them to type.

Reply:
They do not have to specify the host if they have a `~/.conduceconfig` or an email address for that matter.

I cover that in "Getting Started" (README.md in GitHub).

Reply:
Is there a string I could use in place of `--user=new_user@conduce.com` that indicates that the user could enter their email address?  Maybe `--user=<email-address>`

Reply:
If it's already documented elsewhere, I suggest you make a statement indicating something like "the following examples assume you have already configured the system as specified in the getting started guide" (with a link)

And yes to the `--user=<email-address>`